brian2/units/fundamentalunits.py

```diff
@@ -653,11 +651,7 @@ def get_dimensions(obj):
         # The following is not very pretty, but it will avoid the costly
         # isinstance check for the common types
         # Workaround for Python 3 which no longer has a long type
```

Comment:
This single comment line (workaround...) could be removed now.

Reply:
Sorry I think you did this subsequently, was trying to work through comments in order and got confused by the way github works.